index.js

```diff
@@ -1,9 +1,16 @@
 #! /usr/bin/env node
 
+'use strict';
+
 require('dotenv').load({silent: true});
 
+var Promise = require('bluebird');
```

Comment:
'Promise' is defined in a future version of JavaScript. Use a different variable name to avoid migration issues.